### PR TITLE
PIMOB-XXXX: Fix ApplePay token type

### PR DESCRIPTION
**Pull request by ghost**
After some investigations, 
- Fix ApplePay token type to be 'applepay' .
- Add apple pay simulation logic for payment

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Frames Ios?
_Put an `x` in the boxes that apply_

* [ ] Bugfix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

* [ ] Lint and unit tests pass locally with my changes
* [ ] I have added tests that prove my fix is effective or that my feature works
* [ ] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: iOS Example Frame/iOS Example Frame/ViewController/HomeViewController.swift [hunk 1]
```diff
@@ -7,6 +7,7 @@
 //
 
 import Frames
+import PassKit
```
Comment (alex-ioja-yang-cko) [inline iOS Example Frame/iOS Example Frame/ViewController/HomeViewController.swift:10]: Not part of this PR

**Review by alex-ioja-yang-cko — DISMISSED** [review]
Need to undo the HomeViewController changes

File: iOS Example Frame/iOS Example Frame/ViewController/HomeViewController.swift (hunk 1 above)
Comment (alex-ioja-yang-cko) [reply iOS Example Frame/iOS Example Frame/ViewController/HomeViewController.swift:10]: @ehab-al-cko , please review your PR

Comment (deepesh-vasthimal-cko) [reply iOS Example Frame/iOS Example Frame/ViewController/HomeViewController.swift:10]: @alex-ioja-yang-cko I thought adding the PassKit integration in sample app makes sense and improves our sample app. Why do you think we should not do this now ?

**Review by alex-ioja-yang-cko — APPROVED** [review]
